imports/client/components/App.tsx

```diff
@@ -229,7 +240,15 @@ const AppNavbar = () => {
   // correct amount of space in the top bar even if we haven't actually picked
   // a nonempty source for it yet.
   return (
-    <NavbarInset fixed="top" bg="light" variant="light" className="py-0">
+    <NavbarInset
+      fixed="top"
+      variant="light"
+      style={{
+        backgroundColor: "#f0f0f0",
```

Comment:
How attached are you to this darker `backgroundColor`?  I ask because the medium-grey on light grey is worse for text readability in the topbar than the previous `#f8f8f8` (see https://webaim.org/resources/contrastchecker/ ) and I found it a little harder to read the active breadcrumb, which is arguably the most important thing to be able to read in the navbar, since it's how you tell which puzzle you're looking at.

Can we keep this at `#f8f8f8`, or even go full `#fff` (which would pass WCAG AA for normal text)?  Alternately: could we make the active breadcrumb foreground color darker?

If the overall goal here was making the navbar feel sufficiently "separate" from the rest of the page that it's not getting in the way of readability of stuff farther down on e.g. the puzzle list page, another thing we could try to make the navbar easier to ignore/distinguish would be to (probably in a separate PR, the layout stuff is somewhat finicky) make the non-fixed-layout puzzle pages at desktop widths be actually able to scroll away the navbar, rather than leaving it on top at all times, which I think is the difference in behavior that makes our navbar feel more awkwardl than those on some other sites.